docs/REST API/rest-api.md

```diff
@@ -8804,7 +8827,7 @@ Get a list of all software.
     {
       "id": 32,
       "name": "1Password – Password Manager",
-      "software_package": null,
+      "available_for_install": false,
```

Comment:
#### Dev note:
We think that breaking change is ok here. It gives us more flexibility for the future if we want to bring `software_package` as object with information that we have in `GET software/titles/:id`

@rachaelshaw Wdyt?

Reply:
Sounds good, thanks.